tests/surface_attributes/classify_aspect_test.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import rasterio
+import pytest
+from pathlib import Path
+
+from eis_toolkit.exceptions import (
+    InvalidRasterBandException,
+    InvalidParameterValueException,
+)
+from eis_toolkit.surface_attributes.classification import classify_aspect
+from eis_toolkit.surface_attributes.parameters import first_order
+
+parent_dir = Path(__file__).parent
+raster_path_single = parent_dir.joinpath("../data/remote/small_raster.tif")
+raster_path_multi = parent_dir.joinpath("../data/remote/small_raster_multiband.tif")
+raster_path_nonsquared = parent_dir.joinpath("../data/remote/nonsquared_pixelsize_raster.tif")
+
+
+@pytest.mark.parametrize("num_classes", [8, 16])
+def test_aspect_classification(num_classes: int):
+    results_dict = {
```

Comment:
I would move this outside the function definition and with the name `RESULTS_DICT` to signify that it is a module variable.